src/nvim/macros.h

```diff
@@ -154,6 +154,6 @@
 /// zero in those cases (-Wdiv-by-zero in GCC).
 #define ARRAY_SIZE(arr) ((sizeof(arr)/sizeof((arr)[0])) / ((size_t)(!(sizeof(arr) % sizeof((arr)[0])))))
 
-#define RGB(r, g, b) ((r << 16) | (g << 8) | b)
+#define NVIM_RGB(r, g, b) ((r << 16) | (g << 8) | b)
```

Comment:
I forgot to ask earlier, hence the delay.

Anyways, I'm curious: what's the benefit of using a function in this case as opposed to the macro mentioned above? I'm guessing the function provides extra type checking, which would avoid or at least reduce the possibility of left-shifting to the point of undefined behaviour if a signed type were accidentally supplied. As I said, I'm unsure :P